301-eventHub-create-authrule-namespace-and-eventHub/azuredeploy.json

```diff
@@ -0,0 +1,124 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
```

Comment:
Changed to   "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",